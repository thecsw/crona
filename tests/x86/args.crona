manyArgs : void (a : int, b: bool, c: byte, d: int, e: int, f: int, g: int, h: int) {
    write a;
    write "\n";
    write b;
    write "\n";
    write c;
    write "\n";
    write d;
    write "\n";
    write e;
    write "\n";
    write f;
    write "\n";
    write g;
    write "\n";
    write h;
    write "\n";
}

main : void () {
    manyArgs(512, false, 4, 10, 11, 12, 13, 14);
}