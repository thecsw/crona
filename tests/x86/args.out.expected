512
false
4
10
11
12
13
14
